src/commands/validate.js

```diff
@@ -25,14 +47,46 @@ const validate = (context) => {
     })
     .then((errors) => {
       if (errors.length) {
-        context.line(`\nMake any changes to your project and rerun this command.`);
+        context.line('Your app is structurally invalid. Address concerns and run this command again.');
       } else {
-        context.line(`\nThis project looks good!`);
+        context.line('This project looks good!');
```

Comment:
Maybe "structurally good"? I don't like how it sounds, but it also might be strange to see "This project looks good!" and then see some style errors.